test/run-integration-tests.sh

```diff
@@ -47,7 +47,9 @@ declare -A results
 declare -A times
 
 COUNT=0
-FAILURES=0
+COUNT_SUCCESS=0
+COUNT_SKIPPED=0
+COUNT_FAILURES=0
```

Comment:
Imho, this should be called `COUNT_FAILED`. It fits with skipped and rolls better off the tongue.

Reply:
Thanks. Addressed.